cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -751,6 +763,7 @@ context("Test for ActionPopover component", () => {
           alertDialogPreview().should("exist");
         });
 
+      // eslint-disable-next-line jest/valid-expect-in-promise
```

Comment:
same here?

Reply:
As above, regarding the `then...` in the code.